### ISSUE TITLE
Fix `at-rule-no-unknown` false positives for `@scroll-timeline`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#6548]
### What steps are needed to reproduce the bug?

use `@scroll-timeline`
see https://caniuse.com/css-scroll-timeline

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "at-rule-no-unknown": "true"
  }
}
```

### Proposal to fix the bug

edit https://github.com/stylelint/stylelint/blob/main/lib/reference/atKeywords.js

**Comment by ybiquitous** [comment]
@Mouvedia Thanks for the report. Sounds good to me.

I've labeled the issue as **ready to implement**. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.
